src/dbnode/client/types.go

```diff
@@ -78,6 +78,9 @@ type Session interface {
 	// FetchTaggedIDs resolves the provided query to known IDs.
 	FetchTaggedIDs(namespace ident.ID, q index.Query, opts index.QueryOptions) (iter TaggedIDsIterator, exhaustive bool, err error)
 
+	// Aggregate aggregates values from the database for the given set of contraints.
```

Comment:
nit: `constraints`

At what level in the stack do we start referring to this as a `TagCompletion` query or something similar v.s. an Aggregate?